### PR TITLE
102 summarize package scope reference introduction vignette in readme

### LINKED ISSUE #102: Summarize package scope, reference introduction vignette in README
Background: the README is important to give the first time user a quick start into the package.

To do:

- [ ] Add the package scope within the very first section of the README, i.e. just after "The mmrm package implements mixed models for repeated measures (MMRM) in R based on Template Model Builder (TMB).", e.g. bullet list with:
	- [ ] response: normally distributed
	- [ ] covariance structures: unstructured, Toeplitz, AR1, compound symmetry
	- [ ] groups: either single covariance structure for all subjects, or separate by grouping variable
	- [ ] variances: homogeneous or heterogeneous between time points
	- [ ] least square means: `emmeans` package can be used with outputs to obtain least square means
	- [ ] degrees of freedom: Satterthwaite adjustment (planned is to add Kenward-Roger adjustment as alternative)
	- [ ] and others, please see through vignettes for major features
- [ ] Reference the introduction vignette under "Details", next to the other two vignette references.

### PR DESCRIPTION
**Pull request by BrianLang**
closes #102 

Quick summary starting from issue list. 
Updated some phrasing to be more concise. 
Added missing topics.

is the organization reasonable? 
is the *main features* text OK?

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -20,6 +20,23 @@ knitr::opts_chunk$set(
 The `mmrm` package implements mixed models for repeated measures (MMRM) in R
 based on Template Model Builder (TMB).
 
+*Main Features*
```
Comment (danielinteractive) [inline README.Rmd:23]:
```suggestion
## Main Features
```
to use the same styling as below

File: README.Rmd [hunk 2]
```diff
@@ -20,6 +20,23 @@ knitr::opts_chunk$set(
 The `mmrm` package implements mixed models for repeated measures (MMRM) in R
 based on Template Model Builder (TMB).
 
+*Main Features*  
+
+* Responses are assumed normally distributed.
+* Covariances: 
+  * Structures: unstructured, Toeplitz, AR1, compound symmetry, and ante-dependence.
+  * Groups: shared covariance structure for all subjects, or shared by group.
```
Comment (danielinteractive) [inline README.Rmd:28]:
```suggestion
  * Groups: shared covariance structure for all subjects, or group specific covariance structures.
```

File: README.Rmd [hunk 3]
```diff
@@ -20,6 +20,23 @@ knitr::opts_chunk$set(
...
+  * Supports both time-homogeneous and heterogeneous structures.
```
Comment (danielinteractive) [inline README.Rmd:29]:
```suggestion
  * Variances: homogeneous or heterogeneous across time points.
```

File: README.Rmd [hunk 4]
```diff
@@ -20,6 +20,23 @@ knitr::opts_chunk$set(
...
+* Hypothesis testing:
+  * Least square means: `emmeans` package can be used with model outputs to obtain
+least square means.
+  * Satterthwaite-adjusted one- and multi-dimensional contrasts. (future work: Kenward-Roger)
```
Comment (danielinteractive) [inline README.Rmd:33]:
```suggestion
  * Degrees of freedom adjustment: Satterthwaite-adjusted one- and multi-dimensional contrasts.
```

Comment (danielinteractive) [reply README.Rmd:33]: let's maybe take out the KR here

File: README.md [hunk 5]
```diff
@@ -11,6 +11,26 @@ public.](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostat
 The `mmrm` package implements mixed models for repeated measures (MMRM)
 in R based on Template Model Builder (TMB).
 
+*Main Features*
```
Comment (danielinteractive) [inline README.md:14]: please regenerate the md file after changes in Rmd

**Review by danielinteractive — COMMENTED** [review]
thanks @BrianLang, please see few comments inside

File: README.Rmd [hunk 6]
```diff
@@ -20,6 +20,23 @@ knitr::opts_chunk$set(
 The `mmrm` package implements mixed models for repeated measures (MMRM) in R
 based on Template Model Builder (TMB).
```
Comment (danielinteractive) [inline README.Rmd:21]:
```suggestion
The `mmrm` package implements mixed models for repeated measures (MMRM) in R.
```

Comment (danielinteractive) [reply README.Rmd:21]: then it is not confusing anybody, and we can elaborate on this more in the algorithm vignette

**Comment by danielinteractive** [comment]
@BrianLang do you have any question on the PR comments?

**Comment by BrianLang** [comment]
No questions, just on holiday this week without a computer ✌️

**Review by danielinteractive — APPROVED** [review]

**Pull request merged**